src/main/java/io/javarig/util/Utils.java

```diff
@@ -2,7 +2,7 @@
 
 public class Utils {
     public static String getFieldNameFromSetterMethodName(String setterMethodName, String setterPrefix){
-        String fieldName = setterMethodName.replace(setterPrefix,"");
+        String fieldName = setterMethodName.replaceFirst(setterPrefix,"");
```

Comment:
A check to verify that setterMethodName actually starts with setterPrefix seems important to add (though we only pass valid setters)